src/Form.php

```diff
@@ -366,7 +366,7 @@ public function renderView()
      */
     public function ajaxSubmit()
     {
-        $this->_add($cb = new jsCallback(), ['desired_name' => 'submit', 'POST_trigger' => true]);
+        $this->_add($cb = new jsCallback(), ['desired_name' => 'submit', 'postTrigger' => $this->name.'_submit']);
```

Comment:
@ibelar  - can you confirm that form's submission includes a post parameter which is consistent with this naming? Previously we used $callback->name but ATK shortens long names, this way it's more correct.